### PR TITLE
Feature: Family statistics

### PR DESCRIPTION
**Pull request by Dragemil**
Screenshot_20210110_200209

### CODE REVIEW

File: mobile/lib/contracts/tree/contracts.dart [hunk 1]
```diff
@@ -50,6 +50,33 @@ class TreeDTO {
   Map<String, dynamic> toJson() => _$TreeDTOToJson(this);
 }
 
+@JsonSerializable()
+class TreeStatsDTO {
+  final int totalNumberOfPeople;
+  final double averageLifespanInDays;
+  final Map<String, int> numberOfPeopleOfEachGender;
```
Comment (BartoszChrostowski) [inline mobile/lib/contracts/tree/contracts.dart:57]: Can you map enum to int?

File: mobile/lib/features/stats/view/stats_view.dart [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            const StatsLabel("Family members average lifespan:"),
+            Text(
+              "${(stats.averageLifespanInDays / 365.25).toStringAsFixed(2)} yrs.",
+              style: theme.textTheme.headline2,
+              textAlign: TextAlign.center,
+            ),
+          ],
+          const Divider(),
+          LabeledPieChart(
+              label: "Family members genders counts:",
```
Comment (BartoszChrostowski) [inline mobile/lib/features/stats/view/stats_view.dart:115]: As far as I am concerned; "genders count"

File: mobile/lib/features/stats/view/stats_view.dart [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+          const Divider(),
+          const StatsLabel("Family members average count of children:"),
+          Text(
+            "About ${stats.averageNumberOfChildren.toStringAsFixed(2)} children",
+            style: theme.textTheme.headline2,
+            textAlign: TextAlign.center,
+          ),
+          const Divider(),
+          LabeledPieChart(
+              label: "Married and single family members counts:",
```
Comment (BartoszChrostowski) [inline mobile/lib/features/stats/view/stats_view.dart:126]: Same as above "count"

**Review by BartoszChrostowski — CHANGES_REQUESTED** [review]

File: mobile/lib/features/stats/view/stats_view.dart (hunk 2 above)
Comment (Dragemil) [reply mobile/lib/features/stats/view/stats_view.dart:115]: That's also not true, genders count is always the same in our app - 4.

File: mobile/lib/contracts/tree/contracts.dart (hunk 1 above)
Comment (Dragemil) [reply mobile/lib/contracts/tree/contracts.dart:57]: I don't see how it's related

**Pull request merged**